src/Type/InterfaceType/EnqueuedAsset.php

```diff
@@ -52,7 +52,7 @@ public static function register_type( TypeRegistry $type_registry ) {
 				],
 				'version'      => [
 					'type'        => 'String',
-					'description' => __( 'The version of the enqueued asset', 'wp-graphql' ),
+					'description' => __( 'The version of the enqueued asset. Defaults to the current version of WordPress.', 'wp-graphql' ),
```

Comment:
I wonder if we should not disclose this as it could leave sites vulnerable. 

If a security report were announced about a specific version of WordPress and folks were able to identify that any given site was using that version from this query, it would help bad actors identify which sites are vulnerable. 🤔